src/csi-s3/chart/templates/attacher.yaml

```diff
@@ -62,3 +62,16 @@ spec:
             path: {{ $csiS3Properties.sidecars.kubeletPath }}/plugins/csi-s3
             type: DirectoryOrCreate
           name: socket-dir
+      
+      {{- with .Values.nodeSelector }}
+      nodeSelector:
```

Comment:
the `attacher` block is not referenced here. Modify corresponding to changes in `values.yaml`